content/applications/hr/recruitment/new_job.rst

```diff
@@ -44,40 +48,43 @@ options, and then click :guilabel:`Configuration` to edit the details.
    :align: center
    :alt: Edit the job position card.
 
-Enter the job description in the :guilabel:`Job Summary` tab. This information is what is visible to
-potential applicants when searching for available jobs.
-
-Recruitment
-~~~~~~~~~~~
-
 All the basic information about the job position is listed under the :guilabel:`Recruitment` tab.
-None of the fields are required, but it is a good idea to provide at least a few details, such as
-where the job is located.
+
```

Comment:
the image beneath the description of the various data items to enter is the image for the form- it is the new_job/recruitment-tab.png. I thought having it after the explanations of the fields was helpful since they would read the items, then see what it looks like populated on the form. I could move it up but I thought it made sense there.